packages/components/src/Popover/Popover.css

```diff
@@ -0,0 +1,83 @@
+:root {
+  --card--background-color: var(--color-white);
+  --card--base-padding: var(--space-base);
+  --popover--padding: var(--space-base);
+  --popover--width: calc(var(--base-unit) * 21.875);
+  --dismiss--button-padding: calc(var(--base-unit) / 4);
+}
+
+.header {
+  display: flex;
+  position: "relative";
+  padding: var(--popover--padding);
+  justify-content: space-between;
+  align-items: center;
+}
+
+.dismissButton {
+  position: absolute;
+  top: 0;
+  right: 0;
+  padding: var(--dismiss--button-padding);
+}
+
+.popover {
+  z-index: 1;
```

Comment:
Can we use one of our elevation variables here? This feels like it should have the same elevation as `menu`:

```
  --elevation-menu: 6;
```

Although you could also use `elevation-base` as `1` seems to be working here for you.